### ISSUE TITLE
Improve UX of contest reports

### ISSUE THREAD

**Issue by Simon-Busch** [code-423n4/code423n4.com#2116]
## Opportunity:
We have a huge and growing data source of contests reports. As of today, we give access to these reports by contests. 

We could improve this UX by giving some possibilities to make some research in reports, sort the reports with some specific keys.
Example:
-as a user, I would like to see all the High severity findings ...
-as a sure, I would like to be able to search for specific findings

## Desired outcome:
Re-working the UI/UX, add possibility to make some research through these reports, look by type of breaches found … 

## Proposed solution:
- @tealtigrrr to rethink UI ?
- Implement searching system
- Maybe think about some TAGS in the future for people to be able to look up to specific breaches

**Comment by Simon-Busch** [comment]
@tealtigrrr and @sockdrawermoney , I would love to have your opinion on that :)

**Comment by sockdrawermoney** [comment]
I agree this could be nice but following warden auth I think there are some more urgent projects of decent scale that are going to take our attention :)
